src/main/scala/endpoints/CustomerRoutes.scala

```diff
@@ -0,0 +1,36 @@
+package endpoints
+
+import cats.effect.kernel.Concurrent
+import cats.implicits.{toFlatMapOps, toFunctorOps}
+import io.circe.syntax._
+import models.CustomersInterpreter
+import models.Domain.Customer._
+import models.Domain.{Customer, CustomerId}
+import org.http4s.HttpRoutes
+import org.http4s.circe._
+import org.http4s.dsl.Http4sDsl
+import repo.Repository.CustomerRepository
+
+object CustomerRoutes {
+
+  def customerRoutes[F[_]](repository: CustomerRepository)(implicit F: Concurrent[F]): HttpRoutes[F] = {
```

Comment:
`customerRoutes` is just `CustomerController`, it should not have access to `CustomerRepository` because it bypass all business logic (for authorization, checking permission) that resides in `CustomerService`

so `CustomerService` is the only one concept that can directly communicate with specific business repository (customer)

CustomerService **cannot** use other repositories directly even taking into consideration that CustomerService is in domain package - the reason is the same as with controller - bypassing logic of for example ProductService

customerService has fields with only ONE respository: CustomerRepository, and MULTIPLE other **services**